mm/2s2h/Rando/StaticData/Options.cpp

```diff
@@ -14,7 +14,9 @@ namespace StaticData {
 // clang-format off
 std::map<RandoOptionId, RandoStaticOption> Options = {
     RO(RO_ACCESS_DUNGEONS,             RO_ACCESS_DUNGEONS_FORM_AND_SONG),
+    RO(RO_ACCESS_MAJORA_MASKS_COUNT,   0),
     RO(RO_ACCESS_MAJORA_REMAINS_COUNT, 0),
+    RO(RO_ACCESS_MOON_MASKS_COUNT,     20),
```

Comment:
This ought to match the menu's default, which as stated I feel should be 0.
```suggestion
    RO(RO_ACCESS_MOON_MASKS_COUNT,     0),
```

Reply:
Cool makes sense